### PR TITLE
Adding Superplayer & Co

### PR DESCRIPTION
**Pull request by robsondealmeida**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] Company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside.
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @robsondealmeida, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 85724c3).  See below for details:

```
superplayer-co.md: Company title "Superplayer & Co" doesn't match filename (expected ~ "superplayer-and-co.md")

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @robsondealmeida, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 2cf61a0).  See below for details:

```
README.md: Broken link to company "Superplayer & Co": "/company-profiles/superplayer-co.md"
superplayer-and-co.md: No link to company profile from readme

2 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @robsondealmeida, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 4999731). :heavy_check_mark:

File: company-profiles/superplayer-and-co.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Remote status
+
+We are a remote first company with team members spread across many cities in Brazil (also in London).
+We offer Home Office and co-working allowances for remote members like a service subscription that gives them access on many co-work spaces in the main cities in Brazil.
+Our team is from differents regions of Brazil and, as Brazil is a continental country, have a lot of cultural differences that makes our team unique!
+
+
+## Region
+
+Brazil (Brasil), Latin America.
```
Comment (nylen) [inline company-profiles/superplayer-and-co.md]: Do you hire people from outside of Brazil? Most people use the Region field to say a bit about where the company can hire from.

Comment (robsondealmeida) [reply company-profiles/superplayer-and-co.md]: Hi @nylen. Right now we hire only from Brazil and Latin America.
I added more details about it on our company profile Region section.

Thank you.

**Pull request merged**